sk0-back/src/users/user.entity.ts

```diff
@@ -0,0 +1,16 @@
+import { Entity, Column, PrimaryGeneratedColumn } from 'typeorm';
+
+@Entity()
+export class User {
+  @PrimaryGeneratedColumn()
+  id: number;
+
+  @Column()
+  email: string;
```

Comment:
Not sure about the registration scenarios, but I think email should be unique.

Reply:
yes you are right, made the change, thanks